output file:
processed_classes-GPflow_full_weighted_product, _j, k_diag104.json
function:
_full_weighted_product
Error Cases:
ERROR: file or directory not found: /local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/kernels/test_kernels


Pass or Failed: 1

Related Failed Test Cases:
set()

All Test Cases On Generated code:
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.4, pluggy-1.5.0 -- /local/data0/moved_data/publishablew/GPflow/GPflow/venv/bin/python3
cachedir: .pytest_cache
rootdir: /local/data0/moved_data/Organized_benchmark
collecting ... collected 0 items

============================ no tests ran in 0.00s =============================


Final Test Result:
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.4, pluggy-1.5.0 -- /local/data0/moved_data/publishablew/GPflow/GPflow/venv/bin/python3
cachedir: .pytest_cache
rootdir: /local/data0/moved_data/Organized_benchmark
collecting ... collected 0 items

============================ no tests ran in 0.00s =============================


Initial Result:
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.4, pluggy-1.5.0 -- /local/data0/moved_data/publishablew/GPflow/GPflow/venv/bin/python3
cachedir: .pytest_cache
rootdir: /local/data0/moved_data/Organized_benchmark
collecting ... collected 0 items

============================ no tests ran in 0.00s =============================
